build/helper.py

```diff
@@ -177,10 +177,16 @@ def get_method_parameters_snippet(parameters, skip_session_handle, skip_output_p
             snippets.append(x['python_name'])
     return ', '.join(snippets)
 
-def get_function_parameters_snippet(parameters):
-    '''Returns a string suitable for the parameter list of a method given a list of parameter objects'''
+def get_function_parameters_snippet(parameters, sessionName=None):
```

Comment:
should be session_name
Weird that default is different than in other functions. We should make it match or stop using default parameter value everywhere.